R/showcase.R

```diff
@@ -81,7 +81,7 @@ appMetadata <- function(desc) {
 navTabsHelper <- function(files, prefix = "") {
   lapply(files, function(file) {
     with(tags,
-      li(class=if (tolower(file) %in% c("app.r", "server.r")) "active" else "",
+      li(class=if (tolower(file) %in% c("app.r", "server.r", tolower(.globals$appFile))) "active" else "",
```

Comment:
Is there a reason to still have these hardcoded `app.R` and `server.r` included in the list? Would those be covered by `.globals$appFile`?

Reply:
Also does this logic permit multiple files to have class `active`? i.e. if you run an app with `app.R` and `server.R`, I believe `runApp` will give precedence to `server.R`, but this logic would highlight them both as active, right?

May not be a bug worth fixing in this PR, but if that's true we should at least track it IMHO.

Reply:
All good points. At the moment, `.globals$appFile` will only be populated if `shinyAppFile()` is used, but we could also populate it in `shinyAppDir()` to avoid hard-coding and guessing wrong.

Reply:
Done in 4b2e6fc